Some small edits worth considering (first contribution in EMC community) (#58)
EarthMC/earthmc-docs: src/docs/economy.mdx

All town protections are removed when a town falls. Find abandoned towns and loot the riches. Stealing is allowed.

### Voting

> Run the command `/vote` and click the two links provided to you. You will recieve a gold crate for each site you successfully submit a vote to. Currently, you can recieve up to two gold crates per 24-hour day cycle. You can recieve up to 64 gold or a special item reward on each gold crate you earn. Keep in mind that you are able to vote without being online. If you decide to vote while you're not on the server, your award will be given to you next time you sign in and run the command `/cr claim`.

Reviewer: ```suggestion
Run the command `/vote` and click the two links provided to you. You will receive a gold crate for each site you successfully submit a vote to. Currently, you can receive up to two gold crates per 24-hour day cycle. You can receive up to 64 gold or a special item reward on each gold crate you earn. Keep in mind that you are able to vote without being online but that you will not receive any reward.
```

Author: Last sentence: "Remember that you can vote even if you're not online, but you won't receive any rewards."

### Shops

> Open a shop and sell items to other players. The only thing you need is sufficent stock of the item you are trying to sell, competitive pricing, and a good location. Left-click a chest with the item you want to sell and follow the instructions. The most successful shops are built in nation's capitals. Nation capitals have the benefit of attracting a lot of customers through the nation spawn. The shopping districts of capitals are usually one of the main sources of income for nations and players alike. <br /> To learn more about shops, check out the [shops](/shops) page.

Reviewer: ```suggestion
Open a shop and sell items to other players. The only thing you need is sufficent stock of the item you are trying to sell, competitive pricing, and a good location. Left-click a chest with the item you want to sell and follow the instructions. The most successful shops are built in nation capitals, nation capitals have the benefit of attracting a lot of customers through the nation spawn. The shopping districts of capitals are usually one of the main sources of income for nations and players alike. <br /> To learn more about shops, check out the [shops](/shops) page.
```
